Tasks/ArchiveFilesV2/task.json

```diff
@@ -138,6 +138,14 @@
             "argumentFormat": ""
         }
     },
+    "restrictions": {
```

Comment:
Could you please bump task version also?

Reply:
Done